emulators/spim/files/patch-spim_Makefile.diff

```diff
@@ -1,41 +1,19 @@
---- spim/Makefile.orig	2009-01-14 22:29:50.000000000 -0800
-+++ spim/Makefile	2009-01-14 23:24:24.000000000 -0800
-@@ -51,16 +51,16 @@
- # Path of directory that contains documentation:
- DOC_DIR = ../Documentation
+--- Makefile-ori.txt	2018-09-07 07:57:28.000000000 +0800
++++ Makefile	2018-09-07 07:56:09.000000000 +0800
+@@ -66,13 +66,13 @@
  
--
-+prefix = /usr/local
  
  # Full path for the directory that will hold the executable files:
--BIN_DIR = /usr/bin
-+BIN_DIR = $(prefix)/bin
+-BIN_DIR = $(DESTDIR)/usr/bin
++BIN_DIR = $(DESTDIR)/bin
```

Comment:
Sounds wrong. This should be called `prefix` rather than `DESTDIR`. So: `BIN_DIR = $(prefix)/bin`.

Reply:
Personally I prefer a slightly longer, clear and correct patch compared to a patch with less bytes and obfuscated clarity. It's of course up to upstream to make the code more consistent, but if we are patching it anyway, we might just as well patch it properly. Note that there's quite a difference between `$prefix` and `$DESTDIR` and while it might work by accident in this software, mixing up the two would generally generate faulty binaries/libraries.

Reply:
OK,I will adjust those when I am free.